test/core/event_engine/thread_pool_test.cc

```diff
@@ -85,21 +82,6 @@ void ScheduleSelf(ThreadPool* p) {
   p->Run([p] { ScheduleSelf(p); });
 }
 
-TEST(ThreadPoolDeathTest, CanDetectStucknessAtFork) {
```

Comment:
Let's leave this in with a `DISABLED_` prefix to remind us to do something about this in the distant future.

Reply:
Good idea. I've disabled it and added a note about why, and when it can be considered for re-enabling.